config/client-boot.js

```diff
@@ -0,0 +1,27 @@
+import { h, render } from 'preact';
+
+if (process.env.NODE_ENV === 'development') {
+	// enable preact devtools
+	require('preact/debug');
+}
+else if (process.env.ADD_SW && 'serviceWorker' in navigator && location.protocol === 'https:') {
+	// eslint-disable-next-line no-undef
+	navigator.serviceWorker.register(__webpack_public_path__ + 'sw.js');
+}
+
+const interopDefault = m => m && m.default ? m.default : m;
+
+let app = interopDefault(require('app-entry-point'));
+
+if (typeof app === 'function') {
```

Comment:
Can it ever be something else?

Reply:
It could be a VNode, but I don't mind making this file non-generic.

- [ ] remove generic case handling from client-boot